firefox-ios/firefox-ios-tests/Tests/XCUITests/SearchTest.swift

```diff
@@ -188,8 +188,8 @@ class SearchTests: BaseTestCase {
         changeSearchEngine(searchEngine: "Bing")
         changeSearchEngine(searchEngine: "DuckDuckGo")
         changeSearchEngine(searchEngine: "Google")
-        changeSearchEngine(searchEngine: "eBay")
         changeSearchEngine(searchEngine: "Wikipedia")
+        changeSearchEngine(searchEngine: "eBay")
```

Comment:
On iOS 15, eBay raises up the keyboard assuming the user will enter more on the search box, so the hamburger menu isn't visible for the next `changeSearchEngine`. If eBay is at the last, we don't have to worry weather the keyboard is raised or not.